Sources/PrimerSDK/Classes/Core/PrimerHeadlessUniversalCheckout/Managers/Payment Method Managers/RawDataManager.swift

```diff
@@ -26,16 +40,18 @@ protocol PrimerRawDataTokenizationBuilderProtocol {
     var rawData: PrimerRawData? { get set }
 
     init(paymentMethodType: String)
-    func configureRawDataManager(_ rawDataManager: PrimerHeadlessUniversalCheckout.RawDataManager)
+    func configure(withRawDataManager rawDataManager: PrimerHeadlessUniversalCheckout.RawDataManager)
     func makeRequestBodyWithRawData(_ data: PrimerRawData) -> Promise<Request.Body.Tokenization>
     func validateRawData(_ data: PrimerRawData) -> Promise<Void>
 }
 
 extension PrimerHeadlessUniversalCheckout {
 
     public class RawDataManager: NSObject, LogReporter {
-
-        public var delegate: PrimerHeadlessUniversalCheckoutRawDataManagerDelegate?
+        
+        public typealias Delegate = PrimerHeadlessUniversalCheckoutRawDataManagerDelegate
+        
```

Comment:
Thinking out loud, how does namespacing work with typealias's like this - is it accessible as `PrimerHeadlessUniversalCheckout.RawDataManager.Delegate`, or are we pushing a type named `Delegate` out into the namespace?

Reply:
Logically they are the same, but contractually they are different. 

It wasn't my intent to make this public right now ... though I do think we should do this more broadly in the future. But I'll make this case internal 👍 

(typealiases are a nice work around for not being able to have nested protocols)

Reply:
Ah, of course, it's because the property is public - I'll revert for now and we can discuss making this change en masse soon